### ISSUE TITLE
The map frustrates scrolling on the Resistance Map page on iPhone X Safari in portrait

### ISSUE THREAD

**Issue by leehart** [wtchg-kwiatkowski/observatory-web#441]
(Maybe confine map-zoom to in-map-buttons only on finger-scroll devices, and allow finger-scroll to scroll the page.)
![iphonexsafariresistancemapscroll](https://user-images.githubusercontent.com/4256466/49644776-6cb52900-fa11-11e8-83b7-27fb3397454b.png)

**Comment by benjeffery** [comment]
Do you mean the map can't be used as a page scrolling point? This def works on android.

**Comment by leehart** [comment]
That's right: if you swipe your finger up over the map to scroll the page up, you get the screenshot. (The only way to scroll the page is to swipe on the narrow spaces either side of the map.
